roles/ood_user_reg_cloud/templates/user-reg_conf_shib.j2

```diff
@@ -11,7 +11,10 @@ ProxyPassReverse /static http://127.0.0.1:8000/static
         AuthType shibboleth
         ShibRequestSetting requireSession 1
         Require valid-user
```

Comment:
You will need a `RewriteEngine On` here in order to make the rest Rewrite directive to work.